Gems/Atom/Feature/Common/Assets/Materials/Types/StandardMultilayerPBR.materialtype

```diff
@@ -467,183 +462,6 @@
                     "step": 0.1
                 }
             ],
-            "subsurfaceScattering": [
```

Comment:
What happens when you have an existing StandardPbr/MultilayerPBR material that is using sub-surface scattering? I know with something like a texture path, if the texture goes missing the material will fail to load entirely in the material editor and a fail to process, forcing you to manually update the .json. 

If there's not a strong benefit like a clear performance win (that couldn't be accomplished by adding more static shader options to the .shadervariantlist), then it might be better to wait until we have an version upgrade path for materials. Otherwise, be sure to communicate this out as an impactful change that will require people to update their materials by hand.